### PR TITLE
Ogavrikov branch

### PR DESCRIPTION
**Pull request by Eleksor**
Homework 17

### CODE REVIEW

File: src/test/java/school/redrover/OlGavrikovTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package school.redrover;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class OlGavrikovTest {
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/OlGavrikovTest.java:10]: your class should extend BaseTest

File: src/test/java/school/redrover/OlGavrikovTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    @Test
+    public void testSite() {
+        WebDriver driver = new ChromeDriver();
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/OlGavrikovTest.java:14]: you don't need to use driver but getDriver() method

File: src/test/java/school/redrover/OlGavrikovTest.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        driver.manage().window().maximize();
+        driver.get("https://the-internet.herokuapp.com/");
+
+        String titleLoginPage = driver.getTitle();
+        Assert.assertEquals(titleLoginPage, "The Internet");
+
+        WebElement firstClick = driver.findElement(By.xpath("//a[@href=\"/basic_auth\"]"));
+        firstClick.click();
+
+        driver.quit();
```
Comment (tereshenkov29) [inline src/test/java/school/redrover/OlGavrikovTest.java:24]: beforeMethod and AfterMethod are already configured within BaseTest - as a result, you should exctude this line from your code

File: src/test/java/school/redrover/OlGavrikovTest.java (hunk 1 above)
Comment (Eleksor) [reply src/test/java/school/redrover/OlGavrikovTest.java:10]: Ничего про BaseTest не слышал на 17м уроке

File: src/test/java/school/redrover/OlGavrikovTest.java (hunk 3 above)
Comment (Eleksor) [reply src/test/java/school/redrover/OlGavrikovTest.java:24]: Вроде бы просили просто аппрувить друг другу первые pr, вы сбиваете сейчас меня с толку

File: src/test/java/school/redrover/OlGavrikovTest.java (hunk 1 above)
Comment (tereshenkov29) [reply src/test/java/school/redrover/OlGavrikovTest.java:10]: the latest lecture related to the Java Project is 19 but not 17 - something was changed in the process

you MUST watch the record of the 19th lecture and then refactor your code and perform new commit

File: src/test/java/school/redrover/OlGavrikovTest.java (hunk 2 above)
Comment (Eleksor) [reply src/test/java/school/redrover/OlGavrikovTest.java:14]: Это же просто задание на PR((

**Pull request closed without merging**